### DISCUSSION TITLE
How can I contribute?

### DISCUSSION THREAD

**Discussion by dodiego** [apache/age#25] [General] [answered]
I'm Node.js developer and I'm interested in contribute to the node.js client and/or the AgensViewer, how can I proceed?

**Comment by aked21** [comment, marked as answer]
> I'm Node.js developer and I'm interested in contribute to the node.js client and/or the AgensViewer, how can I proceed?

Hello, there is a separate GitHub project for AgensViewer, but AgensViewer is for AgensGraph, a fork of PostgreSQL 

And as for the AGE (on top of Postgres 11) , we (Bitnine) are going to set up a new GitHub repo to develop a new visualization tool similar to AgensViewer (renamed to AGViewer). We will let you know when the repo is ready, probably early next week :)

We appreciate all kinds of contributions but now we need drivers or APIs for various languages like node.js, Go, python ...

We are putting a final touch of creating a user guide for the latest 0.3.0 release. 
We will have the doc available here in a few days.
Hopefully this will give you a better understanding about AGE and also get you ready to develop node.js client

**Reply by dodiego** [reply]
Awesome! 

I'll wait for the new docs, thank you!

**Reply by aked21** [reply]
here attached Apache AGE 0.3.0 doc ... thanks to the AGE team :) 
[Apache AGE (Incubating).pdf](https://github.com/apache/incubator-age/files/6046603/Apache.AGE.Incubating.pdf)

in next few days, will think about the most access-friendly way of making the doc contents available through Readme, Confluence, Apache AGE website, ...
